packages/babel-plugin-transform-parameters/src/params.js

```diff
@@ -60,7 +60,8 @@ export default function convertFunctionParams(path, loose) {
   for (let i = 0; i < params.length; i++) {
     const param = params[i];
 
-    if (param.isAssignmentPattern() && loose) {
+    const paramIsAssignmentPattern = param.isAssignmentPattern();
+    if (paramIsAssignmentPattern && loose) {
```

Comment:
Instead of setting `loose` to true for setters, could you update this check to `(loose || node.kind === "set")`?
It would be a little more descriptive and, if we use `loose` for other parts of the transform on the future, they won't be automatically enabled for setters since they might not be needed.

Reply:
Yep I was thinking that as well!

Reply:
In the beginning i did so. But later..